test/razorpay/test_payment.rb

```diff
@@ -298,6 +298,18 @@ def test_validate_vpa
       stub_post(%r{payments/validate/vpa$}, 'fake_validate_vpa',param_attr.to_json) 
       payment = Razorpay::Payment.validate_vpa param_attr.to_json 
       assert_equal param_attr[:vpa], payment.vpa
-    end  
+    end
+    
+    def test_expendDetails_emi
```

Comment:
please add test case for 5xx response for fetch api

Reply:
We can't mock the response status.

Reply:
Added BAD REQUEST json to test the failure cases